crates/oxc_semantic/src/symbol/mod.rs

```diff
@@ -37,6 +41,8 @@ bitflags! {
         const FunctionScopedVariable  = 1 << 0;
         /// A block-scoped variable (let or const)
         const BlockScopedVariable     = 1 << 1;
+        /// A const variable (const)
+        const ConstVariable           = 1 << 2;
```

Comment:
Does the `ConstVariable`  include import binding?

Reply:
Oh I see. I think we could have separate flags set for import bindings and filter on it in some other rules like no-import-assign.

Reply:
Never mind. I found the no-import-assign.
https://eslint.org/docs/latest/rules/no-import-assign#rule-details

Reply:
Imported bindings should be marked with another flag, TypeScript calls it an `Alias`.